hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -179,6 +179,13 @@
     FRONTEND_STARTUP_TOPIC_SCHEMA_LOADING_RETRY_COUNT("frontend.startup.topic.schema.loading.retry.count", 3),
     FRONTEND_STARTUP_TOPIC_SCHEMA_LOADING_THREAD_POOL_SIZE("frontend.startup.topic.schema.loading.thread.pool.size", 16),
 
+    FRONTEND_PUBLISHING_STARTUP_VALIDATION_ENABLED("frontend.publishing.startup.validation.enabled", false),
```

Comment:
Maybe we should name this like `frontend.kafka.publishing.startup.validation.enabled` ? It'll suggest that it is publishing to kafka itself, not to hermes frontend

Reply:
in this case I suggest` frontend.broker.publishing.startup.validation.enabled`, this will be more provider-agnostic and not coupled to Kafka

Reply:
ok, agree ;)